src/translation/Translation.ts

```diff
@@ -0,0 +1,45 @@
+import trJson from './tr.json'
+
+export class Translation {
+    data: any
+    fallback: any
+
+    constructor(data: any, fallback: any) {
+        if (!data) throw Error('Translation is undefined')
+        if (!fallback) throw Error('Translation fallback is undefined')
+        this.data = data
+        this.fallback = fallback
+    }
+
+    tr(key: string, parameters?: string[]): string {
```

Comment:
maybe call this ```get```? Then one could call this like ```translation.get("bla")```

I find this easier to pronounce when reading it instead of ```translation.tr("bla")```

Reply:
Ok. Just fyi: the idea would be to make the translation object rather short like `t` so that it gets `t.get` as we will have a translation probably in many places.